### ISSUE TITLE
Request to join Backdrop Contributed Project Group 

### ISSUE THREAD

**Issue by alanmels** [backdrop-ops/contrib#372]
I'de like to join Backdrop Contributed Project Group and start contributing extensions to Backdrop.

- I do agree to the Backdrop Contributed Project Agreement.

- Here is the link to the Drupal 7 project that I've just ported to Backdrop: https://github.com/altagrade/searchindex_wipe
- I don't have much contribution yet, just commits on https://github.com/backdrop-contrib/drop and suggestion on https://github.com/backdrop-contrib/ip_blocking/issues/2. At the same time I've submitted bunch of suggestions on various issues for core and contrib modules. 
- Alex Shapka (https://github.com/AlexShapka) can give a recommendation for me.

**Comment by jenlampton** [comment]
Hi @alanmels and thanks for the application, it will be great to have you as part of Backdrop contrib!

>  I do agree to the Backdrop Contributed Project Agreement.

Great, thank you! 

Let's review our checklist:
- [ ] If porting a Drupal project, Maintain the Git history from Drupal 7.
- [x] Include a README.md file that includes license and maintainer information.
- [x] Include a LICENSE.txt file that indicates the contributed code is GPL-2.0 (or in rare cases GPL-3.0)

Everything is in order except the git history. This does look like it was a [Drupal module](https://www.drupal.org/project/searchindex_wipe) before moving to Backdrop, and we want to make sure we keep the Drupal commit history intact. 

The short version is that you need to start from a Git clone of the Drupal project, make the changes necessary for Backdrop on top of that, then push the project up to Github.

The longer version, along with all the necessary Git commands, is available here: https://tag1consulting.com/blog/how-maintain-contrib-modules-drupal-and-backdrop-same-time-part-2#cleaning-up-branches-and-tags

**Comment by alanmels** [comment]
@jenlampton Thanks to your comment I've learned another useful thing today. Now I can see all previous contributors on https://github.com/altagrade/searchindex_wipe/graphs/contributors. Please review the module again.

**Comment by jenlampton** [comment]
Great! Checking again...

>  I do agree to the Backdrop Contributed Project Agreement.

- [x] If porting a Drupal project, Maintain the Git history from Drupal 7.
- [x] Include a README.md file that includes license and maintainer information.
- [x] Include a LICENSE.txt file that indicates the contributed code is GPL-2.0 (or in rare cases GPL-3.0)

That's everything! I'm going to send you an invite to Backdrop Contrib shortly. :)

One thing I often overlook is that the default branch on the GitHub project is `master` but you might want to change that to `1.x-1.x` to save you some headaches later on.

Welcome to contrib!

**Comment by alanmels** [comment]
> One thing I often overlook is that the default branch on the GitHub project is `master` but you might want to change that to `1.x-1.x` to save you some headaches later on.

Done.

> Welcome to contrib!

Thank you very much, @jenlampton!

**Comment by alanmels** [comment]
Forgot to close this.
